### DISCUSSION TITLE
update doc to use Trainer.estimated_stepping_batches

### DISCUSSION THREAD

**Discussion by tshu-w** [Lightning-AI/pytorch-lightning#12579] [Idea pool]
[This tutorial](https://pytorch-lightning.readthedocs.io/en/latest/notebooks/lightning_examples/text-transformers.html?highlight=transformer) should use Lightning 1.6 estimated_stepping_batches rather than cal `total_steps` in `setup`.

**Comment by rohitgr7** [comment]
will be updated :)

thanks for noticing it
